contracts/src/LightClientArbitrum.sol

```diff
@@ -0,0 +1,15 @@
+// SPDX-License-Identifier: UNLICENSED
+
+pragma solidity ^0.8.0;
+
+import { LightClient } from "./LightClient.sol";
+
+interface ArbSys {
+    function arbBlockNumber() external view returns (uint256);
+}
+
+contract LightClientArbitrum is LightClient {
+    function currentBlockNumber() public view virtual override returns (uint256) {
+        return ArbSys(address(uint160(100))).arbBlockNumber();
```

Comment:
just highlighting that we're depending on the fact that the ArbSys address for Arbitrum is always the same, which it should be. However, if it changes we can upgrade the contract. Another option is to make the ArbSys address configurable but that shouldn't be necessary. Unless we're already aware that it's possible/likely for arbitrum (or orbit chains) to change their precompile addresses.